.github/workflows/integration_test.yaml

```diff
@@ -8,11 +8,12 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
     secrets: inherit
     with:
+      channel: 1.26-strict/stable
```

Comment:
I'm guessing this is microk8s, why not use 1.28-strict/stable?

Reply:
Updated! Thank you!